benchmarks/cpp/tensorexpr/bench_approx.cpp

```diff
@@ -366,7 +367,7 @@ static void tanh_caffe2(benchmark::State& state) {
   auto tanh = caffe2::TanhFunctor<caffe2::CPUContext>();
   at::tanh_out(A_t, B_ref);
```

Comment:
Why not change this to `at::cpu::tanh_out` as well?

Reply:
Oh, I just didn't bother changing any of the ones that were there for reference (as opposed to for benchmarking).